.gen/anchore/README.md

```diff
@@ -214,18 +219,23 @@ Class | Method | HTTP request | Description
 
 ## Documentation For Authorization
 
+
+
 ## basicAuth
+
 - **Type**: HTTP basic authentication
 
 Example
+
 ```golang
 auth := context.WithValue(context.Background(), sw.ContextBasicAuth, sw.BasicAuth{
-	UserName: "username",
-	Password: "password",
+    UserName: "username",
```

Comment:
Shouldn't these spaces be a tab?

Reply:
It's generated code readme, doesn't matter IMHO.